storage/src/main/java/zipkin2/storage/kafka/streams/TraceStoreTopologySupplier.java

```diff
@@ -79,99 +78,104 @@ public TraceStoreTopologySupplier(String spansTopicName, List<String> autoComple
   @Override public Topology get() {
     StreamsBuilder builder = new StreamsBuilder();
     builder
+        // Logging disabled to avoid long starting times
         .addStateStore(Stores.keyValueStoreBuilder(
             Stores.persistentKeyValueStore(TRACES_STORE_NAME),
             Serdes.String(),
-            spansSerde))
+            spansSerde).withLoggingDisabled())
+        // Disabling logging to avoid long starting times
         .addStateStore(Stores.keyValueStoreBuilder(
             Stores.persistentKeyValueStore(SPAN_IDS_BY_TS_STORE_NAME),
             Serdes.Long(),
-            spanIdsSerde))
+            spanIdsSerde).withLoggingDisabled())
+        // In-memory as service names are bounded
         .addStateStore(Stores.keyValueStoreBuilder(
-            Stores.persistentKeyValueStore(SERVICE_NAMES_STORE_NAME),
+            Stores.inMemoryKeyValueStore(SERVICE_NAMES_STORE_NAME),
```

Comment:
does this mean on restart there would need to be a replay to rebuild these?

Reply:
Nope, in-memory stores will keep using restoring from changelog topics. 

Only persistent stores, like traces and dependency stores will not resume from changelog but from input topics + store available on local disk.